pages/services/elastic/2.6.0-6.6.1/configuration/index.md

```diff
@@ -14,21 +14,21 @@ render: mustache
+- Health check credentials: If you have X-Pack Security enabled, the health check will use the credentials specified in the configuration for authorization. We recommend that you create a specific {{ model.techName }} user/password for this with minimal capabilities rather than using the default superuser `{{ model.serviceName }}`.
+- Plugins: You can specify other plugins via a comma-separated list of plugin names (such as, “analysis-icu”) or plugin URIs.
+- CPU/RAM/Disk/Heap: These will be specific to your DC/OS cluster and your Elasticsearch use cases. Please refer to {{ model.techName }}’s guidelines for configuration.
 - Node counts: At least one data node is required for the cluster to operate at all. You do not need to use a coordinator node. Learn about Elasticsearch node types [here](https://www.elastic.co/guide/en/elasticsearch/reference/current/modules-node.html). There is no maximum for node counts.
-- Master transport port: You can pick whichever port works for your DC/OS cluster. The default is 9300. If you want multiple master nodes from different clusters on the same host, specify different master HTTP and transport ports for each cluster. If you want to ensure a particular distribution of nodes of one task type (e.g., master nodes spread across multiple racks, data nodes on one class of machines), specify this via the Marathon placement constraint.
-- Serial vs Parallel deployment. By default, the DC/OS Elastic Service tells DC/OS to install everything in parallel. You can change this to serial in order to have each node installed one at a time.
-- Serial vs Parallel update. By default, the DC/OS Elastic Service tells DC/OS to update everything serially. You can change this to parallel in order to have each node updated at the same time. This is required, for instance, when you turn X-Pack Security on or off.
-- Custom YAML can be appended to `elasticsearch.yml` on each node
+- Master transport port: You can pick whichever port works for your DC/OS cluster. The default is 9300. If you want multiple master nodes from different clusters on the same host, specify different master HTTP and transport ports for each cluster. If you want to ensure a particular distribution of nodes of one task type (such as master nodes spread across multiple racks, data nodes on one class of machines), specify this via the Marathon placement constraint.
+- Serial vs Parallel deployment. By default, the DC/OS {{ model.techName }} Service tells DC/OS to install everything in parallel. You can change this to serial in order to have each node installed one at a time.
+- Serial vs Parallel update. By default, the DC/OS {{ model.techName }} Service tells DC/OS to update everything serially. You can change this to parallel in order to have each node updated at the same time. This is required, for instance, when you turn X-Pack Security on or off.
+- A custom YAML file can be appended to `elasticsearch.yml` on each node
```

Comment:
It's not a file, it's a configuration setting. The previous phrasing is more accurate.

Reply:
Thanks!